activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-liquibase/src/main/resources/config/query/liquibase/changelog/10-alter.oracle.schema.7.1.0-m17.sql

```diff
@@ -1,2 +1,14 @@
 alter table process_definition
     add category varchar(255);
+
+declare max_process_variable number;
+begin
+    select nvl(max(value), 0) + 1 into max_process_variable from process_variable;
```

Comment:
Shouldn't it be `max(id)` instead of `max(value)`?

Reply:
fixed